src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -27,7 +27,7 @@ const (
 
 	defaultQemuMachineType = QemuPC
 
-	defaultQemuMachineOptions = "accel=kvm,kernel_irqchip"
+	defaultQemuMachineOptions = "accel=kvm,kernel_irqchip,nvdimm"
```

Comment:
This should be a separate change - I don't see how it's inherently related to encrypted guest support.

Reply:
@sandeepg-gupta do you know if `nvdimm` is necessary?